source/tutorial/acceptance-test.md

```diff
@@ -42,18 +42,18 @@ Since we haven't added any functionality to our application yet, we'll use this
 
 To do that, replace occurrences of `/list-rentals` in the generated test with `/`. The test will start our app at the base url, `http://localhost:4200/`, and then do a basic check that the page has finished loading and that the url is what we want it to be.
 
-```/tests/acceptance/list-rentals-test.js{-6,+7,-8,+9,-12,+13}
-import { test } from 'qunit';
-import moduleForAcceptance from 'super-rentals/tests/helpers/module-for-acceptance';
-
-moduleForAcceptance('Acceptance | list-rentals');
-
-test('visiting /list-rentals', function(assert) {
-test('visiting /', function(assert) {
-  visit('/list-rentals');
-  visit('/');
-
-  andThen(function() {
+```/tests/acceptance/list-rentals-test.js{-8,+9,-10,+11,-12,+13}
+import { module, test } from 'qunit';
+import { visit, currentURL } from '@ember/test-helpers';
+import { setupApplicationTest } from 'ember-qunit';
```

Comment:
I'd suggest putting this line right below the qunit import to group them a little better